codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameWriter.java

```diff
@@ -583,7 +583,7 @@ private ChannelFuture writeContinuationFrames(ChannelHandlerContext ctx, int str
                     // The frame header is different for the last frame, so re-allocate and release the old buffer
                     flags = flags.endOfHeaders(true);
                     buf.release();
-                    buf = ctx.alloc().buffer(CONTINUATION_FRAME_HEADER_LENGTH);
+                    buf = ctx.alloc().buffer(FRAME_HEADER_LENGTH);
```

Comment:
Shouldn't we better adjust `CONTINUATION_FRAME_HEADER_LENGTH` to be `==` `FRAME_HEADER_LENGTH`

Reply:
> Shouldn't we better adjust `CONTINUATION_FRAME_HEADER_LENGTH` to be `==` `FRAME_HEADER_LENGTH`

Yes, the first modification was to CONTINUATION_FRAME_HEADER_LENGTH. The check failed at that time, and there was concern about affecting other callers, but upon careful consideration, adjust CONTINUATION_FRAME_HEADER_LENGTH would be better

Reply:
> > Shouldn't we better adjust `CONTINUATION_FRAME_HEADER_LENGTH` to be `==` `FRAME_HEADER_LENGTH`
> 
> Yes, the first modification was to CONTINUATION_FRAME_HEADER_LENGTH. The check failed at that time, and there was concern about affecting other callers, but upon careful consideration, adjust CONTINUATION_FRAME_HEADER_LENGTH =  FRAME_HEADER_LENGTH would be better